README.md

```diff
@@ -636,6 +656,157 @@ When defining an Audience, users and groups can be excluded from the audience. T
+| `Seed` | The value which percentage calculations for `Percentile` are based on. The percentage calculation for a specific user will be the same across all features if the same `Seed` value is used. If no `Seed` is specified, then a default seed is created based on the feature name. |
+
+In the above example, if the feature is not enabled, `GetVariantAsync` would return the variant allocated by `DefaultWhenDisabled`, which is `Small` in this case. 
+
+If the feature is enabled, the feature manager will check the `User`, `Group`, and `Percentile` allocations in that order to allocate a variant for this feature. If the user being evaluated is named `Marsha`, in the group named `Ring1`, or the user happens to fall between the 0 and 10th percentile calculated with the given `Seed`, then the specified variant is returned for that allocation. In this case, all of these would return the `Big` variant. If none of these allocations match, the `DefaultWhenEnabled` variant is returned, which is `Small`.
+
+Allocation logic is similar to the [Microsoft.Targeting](./README.md#MicrosoftTargeting) feature filter, but there are some parameters that are present in targeting that aren't in allocation, and vice versa. The outcomes of targeting and allocation are not related.
+
+### Overriding Enabled State with a Variant
+
+You can use variants to override the enabled state of a feature flag. This gives variants an opportunity to extend the evaluation of a feature flag. If a caller is checking whether a flag that has variants is enabled, then variant allocation will be performed to see if an allocated variant is set up to override the result. This is done using the optional variant property `StatusOverride`. By default, this property is set to `None`, which means the variant doesn't affect whether the flag is considered enabled or disabled. Setting `StatusOverride` to `Enabled` allows the variant, when chosen, to override a flag to be enabled. Setting `StatusOverride` to `Disabled` provides the opposite functionality, therefore disabling the flag when the variant is chosen. A feature with a `Status` of `Disabled` cannot be overridden.
+
+```
+"Allocation": {
+    "Percentile": [{
+        "Variant": "On",
+        "From": 10,
+        "To": 20
+    }],
+    "DefaultWhenEnabled":  "Off",
+    "Seed": "Enhanced-Feature-Group"
+},
+"Variants": [
+    { 
+        "Name": "On",
+        "ConfigurationValue": true
+    },
+    { 
+        "Name": "Off",
+        "ConfigurationValue": false,
+        "StatusOverride": "Disabled"
+    }    
+],
+"EnabledFor": [ 
+    { 
+        "Name": "AlwaysOn" 
+    } 
+] 
+```
+
+In the above example, the feature is enabled by the `AlwaysOn` filter. If the current user is in the calculated percentile range of 10 to 20, then the `On` variant is returned. Otherwise, the `Off` variant is returned and because `StatusOverride` is equal to `Disabled`, the feature will now be considered disabled.

```

Comment:
I hope we can explain a little more about the scenarios where this property can be useful. Can we add a sentence like below at the beginning of this paragraph? 

If you are using a feature flag with binary variants, the `StatusOverride` property can be very helpful. It allows you to keep using APIs like `IsEnabledAsync` and `FeatureGateAttribute` in your application, all while benefiting from the new features that come with variants, such as percentile allocation and seed.

Reply:
I think the example should be immediately followed by its explanation, could we add something similar to this sentence just before the example maybe, In a separate paragraph from the introduction?

Reply:
Sounds good. That works.